api/scripts/swagger.go

```diff
@@ -36,8 +36,8 @@ func main() {
 			reader := bufio.NewReader(f)
 			writer := bufio.NewWriter(out)
 
-			_, _ = writer.WriteString("package api\n\nfunc init() {\n")
-			_, _ = writer.WriteString("	Swagger.Add(\"" + name + "\", `")
+			_, _ = writer.WriteString("package api\n\nfunc init() {\n")   // nolint: errcheck
+			_, _ = writer.WriteString("	Swagger.Add(\"" + name + "\", `") // nolint: errcheck
```

Comment:
💭 if we add the `// nolint` comment, can't we just write 
```suggestion
			writer.WriteString("	Swagger.Add(\"" + name + "\", `") // nolint: errcheck
```

Reply:
Nope, it'll still complain that the function returns type `Error` that we aren't checking.

Reply:
OK, this is confusing. A) How does [this file pass then](https://github.com/chef/automate/pull/608/files#diff-a099af9c4237c8d4028c5612518211c0R52)? B) Why is this, not living in `components/automate-gateway` even included in linting?